rules/helm/package.sh

```diff
@@ -4,15 +4,18 @@ set -o errexit -o nounset
 
 build_dir="tmp/build/${PACKAGE_DIR}"
 mkdir -p "${build_dir}"
+
 cp -L -R "${PACKAGE_DIR}"/* "${build_dir}"
 
 for t in ${TARS}; do
-  tar xf "${t}" -C "${build_dir}"
+  tar xf "${t}" -C "${build_dir}"  > /dev/null
 done
 
+echo "operatorChartUrl: \"${OPERATOR_URL}\"" >> "${build_dir}"/Metadata.yaml
```

Comment:
Seems slightly off to put the very-much-kubecf-chart stuff into a generic-looking helm chart creation rule.

Given that `Metadata.yaml` doesn't otherwise exist (and doesn't appear to be a standard thing in either [v2] or [v3]), wouldn't it make sense to have a separate rule to create it in `//deploy/helm/kubecf`, and just add it as an extra file on the `//rules/helm` layer?  Especially since it already takes `srcs`.

Bikeshed, feel free to ignore: `Metadata.yaml` vs `metadata.yaml` vs…

[v2]: https://v2.helm.sh/docs/developing_charts/#the-chart-file-structure
[v3]: https://helm.sh/docs/topics/charts/#the-chart-file-structure

Reply:
> Seems slightly off to put the very-much-kubecf-chart stuff into a generic-looking helm chart creation rule.
> 
> Given that `Metadata.yaml` doesn't otherwise exist (and doesn't appear to be a standard thing in either [v2](https://v2.helm.sh/docs/developing_charts/#the-chart-file-structure) or [v3](https://helm.sh/docs/topics/charts/#the-chart-file-structure)), wouldn't it make sense to have a separate rule to create it in `//deploy/helm/kubecf`, and just add it as an extra file on the `//rules/helm` layer? Especially since it already takes `srcs`.

Yes `Metadata.yaml` doesn't exists in helm,  there is no other way around to add a custom value to the chart directly from the helm cli as far as I can tell.

I don't know, to me looks like adding a layer of over-complexity, because I would expect all the chart metadata needed for packaging in  the `helm` rule, as we already specify (hardcoded): 
```
   chart_name = "kubecf",
    chart_version = "3.0.0",
    app_version = "2.0",
```
 in the same section. But I will try to have a look

> 
> Bikeshed, feel free to ignore: `Metadata.yaml` vs `metadata.yaml` vs…

I picked capital letter just to be coherent with `Chart.yaml` which is already there, but no strong opinion here.

Reply:
Should be fixed in https://github.com/SUSE/kubecf/pull/149/commits/1e7d013029268c6cd42d44b0fd15ee66a270f3b0, I've also added a macro to generate the metadata and added the extra file in the helm layer